SlackAPI/SlackClient.cs

```diff
@@ -878,6 +878,22 @@ public void JoinDirectMessageChannel(Action<JoinDirectMessageChannelResponse> ca
 
             APIRequestWithToken(callback, parameters.ToArray());
         }
+        
+        public void AddPin(
```

Comment:
This should return the same type as the Async, `PinAddedResponse`

I don't remember but wasn't there a generic success type? 🤔